src/cljs/rems/administration/create_catalogue_item.cljs

```diff
@@ -230,7 +230,10 @@
         item-selected? #(= (:form/id %) (:form/id selected-form))
         language @(rf/subscribe [:language])]
     [:div.form-group
-     [:label.administration-field-label {:for form-dropdown-id} (text :t.administration/form)]
+     [:label.administration-field-label {:for form-dropdown-id}
+      (text :t.administration/form)
+      " "
+      (text :t.administration/optional)]
```

Comment:
alternatively this could use localization helper, for example in `rems.administration.create-workflow/workflow-disable-commands-field`:
```clojure
(text-format :t.label/optional (text :t.create-workflow/disable-commands))
```

Reply:
Yeah, that wasn't used in this file yet. I'll add a task to use it (everywhere).